src/components/display/fewlines/Security/Security.tsx

```diff
@@ -0,0 +1,40 @@
+import Link from "next/link";
+import React from "react";
+import styled from "styled-components";
+
+import { RightChevron } from "../Icons/RightChevron/RightChevron";
+import { NeutralLink } from "../NeutralLink/NeutralLink";
+import { ShadowBox } from "../ShadowBox/ShadowBox";
+
+type SecurityProps = {
+  isPasswordSet: boolean;
+};
+
+export const Security: React.FC<SecurityProps> = ({ isPasswordSet }) => {
+  return (
+    <ShadowBox>
+      <Link href={`/account/security/${isPasswordSet ? "update" : "set"}`}>
```

Comment:
Why do we need two URLs?
Is the answer, we did but we don't anymore? 😇

Reply:
Daaaamn nice catch. I chose to default it as `update`.

Reply:
dbb0af5